docs/recipes/recipe-content-analytics/index.md

```diff
@@ -27,7 +27,10 @@ We have already created a custom `content` entity for you in [Iglu Central](http
 
 Snowplow uses self-describing JSON schemas to structure events and entities so that they can be validated in the pipeline and loaded into tidy tables in the warehouse. You can learn more about these data structures [here](/docs/understanding-your-pipeline/schemas/index.md), and about why we take this approach [here](https://snowplowanalytics.com/blog/2020/01/24/re-thinking-the-structure-of-event-data/).
 
-While Try Snowplow only ships with a pre-designed set of custom events and entities required for the recipes, Snowplow BDP lets you create an unlimited number of your own via the [Data Structures UI](/docs/understanding-tracking-design/managing-your-data-structures/ui/index.md) (and API) for Enterprise and via [the Data Structures Builder](/docs/understanding-tracking-design/managing-your-data-structures/ui/builder/index.md) for Cloud. 
+Snowplow BDP expands its capabilities beyond the pre-designed custom events and entities required for the recipes in Try Snowplow. The platform introduces the [Data Structures UI](/docs/understanding-tracking-design/managing-your-data-structures/ui/index.md), featuring the [Data Structures Builder](/docs/understanding-tracking-design/managing-your-data-structures/ui/builder/index.md). This Builder streamlines the creation process of events and entities with a user-friendly, guided setup and automated versioning. 
```

Comment:
Could you make this a reusable block? Not critical but prefer to avoid repetition. It could live inside `docs/recipes`, e.g. `docs/recipes/_custom-events-entities.md`. (See the README for how to use reusable blocks.)